unity-client/Assets/ABConverter/Config.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.IO;
+using UnityEngine;
+
+namespace DCL
+{
+    public abstract partial class ABConverter
+    {
+        public static class Config
+        {
+            internal const string CLI_VERBOSE = "verbose";
+            internal const string CLI_ALWAYS_BUILD_SYNTAX = "alwaysBuild";
+            internal const string CLI_KEEP_BUNDLES_SYNTAX = "keepBundles";
+            internal const string CLI_BUILD_SCENE_SYNTAX = "sceneCid";
+            internal const string CLI_BUILD_PARCELS_RANGE_SYNTAX = "parcelsXYWH";
+            internal const string CLI_SET_CUSTOM_BASE_URL = "baseUrl";
+
+            internal const string CLI_SET_CUSTOM_OUTPUT_ROOT_PATH = "output";
+
+            internal static string ASSET_BUNDLE_FOLDER_NAME = "AssetBundles";
+            internal static string DOWNLOADED_FOLDER_NAME = "_Downloaded";
+
+            internal static char DASH = Path.DirectorySeparatorChar;
```

Comment:
What are the benefits of wrapping a `Path.DirectorySeparatorChar`?

Reply:
Did it just to save a few characters below.